backend/entities/office_hours/event_entity.py

```diff
@@ -77,7 +77,7 @@ def from_model(cls, model: OfficeHoursEvent) -> Self:
             type=model.type,
             description=model.description,
             location_description=model.location_description,
-            date=model.date,
+            date=model.event_date,
```

Comment:
I was getting a type error when running the reset demo script, as the variable name and type name were the same. Changed the name to `event_date` to avoid future problem with naming